
# j#pos: position of the #th joint
float32 j0pos
float32 j1pos
float32 j2pos
float32 j3pos
float32 j4pos
float32 j5pos
string name
# Optional: Fraction of fastest speed the arm can move
float32 speed_ratio
#Optional: Audio file delay
bool wait
